docs/how-to-upload-manifest.md

```diff
@@ -1,107 +1,92 @@
 # How To Push Operator Manifest(s) To App Registry
+
 This document shows how one can push operator manifest(s) into `quay.io` or any other compatible app registry server.
 
-*The example script(s) in this document refers to quay.io*
+*The example scripts in this document refer to quay.io*
 
 ## Format of Operator Manifest
 *As proof of concept, we are storing the operator manifest into one file. It could change in the future.*
 
-The operator manifest file has the following structure -
-```yaml
-name: certified operators
-publisher: redhat
+The operator manifest is stored in a `ConfigMap` with the following structure:
```

Comment:
@tkashem is this true?

Reply:
Seems that way - [here](https://github.com/tkashem/operators) is a repo with the manifests that we have pushed to quay

Reply:
Sure but does it **need** to be in the same format as a `ConfigMap`? I suspect it is being ignored when pulled down by the OperatorSource handler. @tkashem can confirm.

Reply:
Yes, it is being ignored, we are only interested in the `data` section.

Reply:
Given that, @SamiSousa please reword this appropriately and fix the structure.

Reply:
So the file can be formatted as follows:
```yaml
data:
  customResourceDefinitions: |-
  clusterServiceVersions: |-
  packages: |-
```
Does that look right?

Reply:
That looks right to me. @tkashem please confirm.